ClientApp/src/modules/authorization/views/CreateUserView/CreateUserView.tsx

```diff
@@ -3,20 +3,36 @@ import React, {useState} from 'react';
 import {Link, useHistory} from 'react-router-dom';
 import {AuthenticationRoutes, ClientRoutes} from '../../../../routing/routes';
 import {createUser} from '../../helpers/AuthService';
-import {button, errormsg, input} from '../AuthorizationView/AuthorizationView.styles';
+import {button, boxswt, errormsg, input, swt} from '../AuthorizationView/AuthorizationView.styles';
+import { Switch } from "@chakra-ui/react"
+import { CLIENT_TYPE, WORKSHOP_EMPLOYEE } from '../../helpers/AuthService.types';
 
 const CreateUserView: React.FC = ({children}) => {
     const history = useHistory();
     const [username, setUsername] = useState('');
     const [password, setPassword] = useState('');
+    const [name, setName] = useState('');
+    const [surname, setSurname] = useState('');
     const [first, setFirst] = useState(false);
     const [confirmPassword, setConfirmPassword] = useState('');
+    const [type, setType] = useState(false) // CLIENT = false
+    const [telephone, setTelephone] = useState('')
 
     function onChangeUsername(e: React.FormEvent<HTMLInputElement>) {
         setUsername(e.currentTarget.value);
         setFirst(true);
     }
 
+    function onChangeName(e: React.FormEvent<HTMLInputElement>) {
```

Comment:
Formik will simplify for sure

Reply:
explain it, if its not costly I'll think it over

Reply:
Will be costly, but more understandable. Maybe in future refactors...